pkg/Microsoft.Windows.Compatibility/Microsoft.Windows.Compatibility.pkgproj

```diff
@@ -68,10 +68,6 @@
   <ItemGroup>
     <Dependency Include="@(LibraryPackage)" />
     <Dependency Include="@(PrereleaseLibraryPackage)" />
-    <!-- Include dependency to shims package -->
-    <Dependency Include="Microsoft.Windows.Compatibility.Shims">
```

Comment:
This is the logical conclusion of the shiproom message that we aren't going to service 2.0 from 2.1 branch.  The implication is neither will we do so from the vNext branch.  I suppose we could keep this around and apply a better condition to it: making it only apply for nca2.0.

Reply:
Yeah I think we should otherwise folks will not ever know they need to add this reference. 

>This is the logical conclusion of the shiproom message that we aren't going to service 2.0 from 2.1 branch.

Can you clarify this I must have missed that conclusion.

Reply:
> Can you clarify this I must have missed that conclusion.

Since 2.1 has the shims inbox and the shims will continue to ship inbox on later TFMs, they didn't want to take a fix in the 2.1 branch to fix a 2.0 behavior that has been out there for already 4-6 months and no customer has reported it. So the conclusion was to fix this in master, break the dependency for 2.1 and ship a shims package people hitting this issues in 2.0 could use as a workaround and don't ship a compat pack from the 2.1 branch anymore. However I think we should port this to 2.1 as 2.2 is going to branch off 2.1 right?

Reply:
>However I think we should port this to 2.1 as 2.2 is going to branch off 2.1 right?

Not clear yet.